src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx

```diff
@@ -108,21 +114,29 @@ class OrganizationProjects extends AsyncView {
             })}
           </PanelHeader>
           <PanelBody css={{width: '100%'}}>
-            {sortProjects(projectList).map((project, i) => (
-              <PanelItem p={0} key={project.id} align="center">
-                <Box p={2} flex="1">
-                  <ProjectListItem project={project} organization={organization} />
-                </Box>
-                <Box w={3 / 12} p={2}>
-                  <ProjectStatsGraph
-                    key={project.id}
-                    project={project}
-                    stats={projectStats[project.id]}
-                  />
-                </Box>
-              </PanelItem>
-            ))}
-            {projectList.length === 0 && (
+            {projectList ? (
+              sortProjects(projectList).map(project => (
+                <PanelItem p={0} key={project.id} align="center">
+                  <Box p={2} flex="1">
```

Comment:
Same thing with these `Box`

You could maybe even make `PanelItem` a css grid.